corehq/apps/users/tests/test_migrate_roles_to_sql.py

```diff
@@ -237,7 +238,9 @@ def tearDownClass(cls):
             role.delete()
 
     def test_command(self):
+        self.assertTrue(sync_to_couch_enabled(SQLUserRole))
```

Comment:
I didn't grasp on the first read that this disables couch syncing for the entire class. Doesn't that mean if someone saves a user role in the UI while the population command is running, there's a chance that role doesn't get synced to sql?

Reply:
It's using a thread local so it will only disable it for the running thread and not for any other threads. In the case of the management command it's particularly safe since it runs in a completely separate process from the web workers.

Reply:
thanks, that makes sense now💡